composed_configuration/_allauth_support/createsuperuser.py

```diff
@@ -55,7 +55,7 @@ class Meta(User.Meta):
 
     # Don't include "email" in "REQUIRED_FIELDS", to prevent adding that field twice to the
     # "createsuperuser.Command" argument parser
-    REQUIRED_FIELDS: List[str] = []
+    REQUIRED_FIELDS: List[str] = []  # type: ignore
```

Comment:
```suggestion
    REQUIRED_FIELDS: ClassVar[List[str]] = []
```

You'll have to add the appropriate import from `typing` too.